src/policy/rbf.h

```diff
@@ -19,9 +19,9 @@
 class CFeeRate;
 class uint256;
 
-/** Maximum number of transactions that can be replaced by BIP125 RBF (Rule #5). This includes all
+/** Maximum number of transactions that can be replaced by RBF Rule #5. This includes all
```

Comment:
I think it was easier to parse with the `()` kept.

Reply:
Thanks, done in #25902